src/app/domain/teams/guards.py

```diff
@@ -62,4 +62,5 @@ def requires_team_ownership(connection: ASGIConnection, _: BaseRouteHandler) ->
         return
     if any(membership.team.id == team_id and membership.is_owner for membership in connection.user.teams):
         return
-    raise PermissionDeniedException("Insufficient permissions to access team.")
+    msg = "Insufficient permissions to access team."
```

Comment:
Is assigning to msg instead of returning string directly , to make line shorter?

Reply:
It is a ruff rule

Reply:
This is a ruff change.